tests/ColorCoderTest.h

```diff
@@ -0,0 +1,47 @@
+#ifndef COLOR_CODER_TEST_H
+#define COLOR_CODER_TEST_H
+
+#include <iostream>
+#include <cassert>
+#include "../src/ColorCoder.h"
+#include "../src/SimpleColorFormatter.h"
+
+namespace TelCoColorCoder {
+
+    void testNumberToPair(int pairNumber, MajorColor expectedMajor, MinorColor expectedMinor) {
+        ColorPair colorPair = ColorCoder::GetColorFromPairNumber(pairNumber);
+        SimpleColorFormatter formatter;
+        std::cout << "Got pair " << formatter.formatColorPair(colorPair.getMajor(), colorPair.getMinor()) << std::endl;
+        assert(colorPair.getMajor() == expectedMajor);
+        assert(colorPair.getMinor() == expectedMinor);
+    }
+
+    void testPairToNumber(MajorColor major, MinorColor minor, int expectedPairNumber) {
+        int pairNumber = ColorCoder::GetPairNumberFromColor(major, minor);
+        SimpleColorFormatter formatter;
+        std::cout << "Got pair number " << pairNumber << " for "
+                  << formatter.formatColorPair(major, minor) << std::endl;
+        assert(pairNumber == expectedPairNumber);
+    }
+
+    // New function to print the reference manual
+    void printReferenceManual() {
```

Comment:
By keeping `printReferenceManual` along with the tests, we make it impossible for it to be used by others. It can only be run with the tests, not by someone who just wants to print the reference manual.
You have already separated tests into its own folder. Why not separate new functionality into its own file, along with the rest of the production code?